Makefile

```diff
@@ -88,18 +90,18 @@ build-windows-amd64:
 # tool-related commands
 .PHONY: install-tools
 install-tools:
-	go install github.com/client9/misspell/cmd/misspell@v0.3.4
-	go install github.com/google/addlicense@v1.1.1
-	go install github.com/mgechev/revive@v1.3.7
-	go install github.com/open-telemetry/opentelemetry-collector-contrib/cmd/mdatagen@v0.96.0
-	go install github.com/securego/gosec/v2/cmd/gosec@v2.18.2
+	cd $(TOOLS_MOD_DIR) && go install github.com/client9/misspell/cmd/misspell@v0.3.4
+	cd $(TOOLS_MOD_DIR) && go install github.com/google/addlicense@v1.1.1
+	cd $(TOOLS_MOD_DIR) && go install github.com/mgechev/revive@v1.3.7
+	cd $(TOOLS_MOD_DIR) && go install github.com/open-telemetry/opentelemetry-collector-contrib/cmd/mdatagen@v0.96.0
+	cd $(TOOLS_MOD_DIR) && go install github.com/securego/gosec/v2/cmd/gosec@v2.18.2
 # update cosign in release.yml when updating this version
 # update cosign in docs/verify-signature.md when updating this version
-	go install github.com/sigstore/cosign/cmd/cosign@v1.13.1
-	go install github.com/uw-labs/lichen@v0.1.7
-	go install github.com/vektra/mockery/v2@v2.42.0
-	go install golang.org/x/tools/cmd/goimports@latest
-	go install github.com/goreleaser/goreleaser@v1.22.1
+	cd $(TOOLS_MOD_DIR) && go install github.com/sigstore/cosign/cmd/cosign@v1.13.1
+	cd $(TOOLS_MOD_DIR) && go install github.com/uw-labs/lichen@v0.1.7
+	cd $(TOOLS_MOD_DIR) && go install github.com/vektra/mockery/v2@v2.42.0
+	cd $(TOOLS_MOD_DIR) && go install golang.org/x/tools/cmd/goimports@latest
+	cd $(TOOLS_MOD_DIR) && go install github.com/goreleaser/goreleaser@v1.22.1
```

Comment:
I think we want to get rid of the version tags here so the `go.mod` versions get used